library/mt_ip_firewall.py

```diff
@@ -55,15 +55,15 @@
     password:  "{{ mt_pass }}"
     state:     present
     parameter: filter
-    settings:
+    rule:
```

Comment:
is this a fix or a potential improvement?

Reply:
That change was included by mistake as it should not be part of the PR. This change is because the modules is not using "settings" but "rule" as the dictionary that contains the elements to be added/removed to the module. This should be part of another PR for issue #14